src/transaction/log_tran_table.c

```diff
@@ -4011,12 +4009,12 @@ logtb_complete_mvcc (THREAD_ENTRY * thread_p, LOG_TDES * tdes, bool committed)
     }
   else
     {
-#if defined(SA_MODE)
       if (committed && logtb_tran_update_all_global_unique_stats (thread_p) != NO_ERROR)
 	{
 	  assert (false);
 	}
-#else	/* !SA_MODE */	       /* SERVER_MODE */
+#if defined(SERVER_MODE)
```

Comment:
The comment below is explicitly saying that only the stat for db_serial has to be updated in a non-mvcc trx. If you conclude that it is not true by analysis, we have to fix the code below too. In the changed code, it seems that the stat for db_serial already reflected in `logtb_tran_update_all_global_unique_stats()`.

Share your analysis about it.

Reply:
In the previous code, only the stat for db_serial is reflected if no MVCCID is generated. However, in the changed code, all entries are iterated in the stat hash, and It could impact normal transaction processing.

Reply:
I don't know the reason why the reflection code for db_serial is applied in only the "SERVER_MODE".
I left the codes that reflect the serial key in SERVER_MODE since I can not figure out the intention.
I did it to avoid modifying the existing code as much as possible to avoid unintended side effects.

Reply:
No, the stat for db_serial is also reflected in SA_MODE. Only the stat for db_serial is reflected, but we are changing to reflect others too. Do I misunderstand?

Reply:
http://jira.cubrid.org/browse/CUBRIDSUS-17437
The db_serial-related stats reflection under SERVER_MODE is added at the above issue for patching "checkdb" bug.
However, "show index header" statement also has a bug not reflects the stats.
So, I think that the SERVER_MODE limitation and db_serial-related stats reflect code should be removed.